src/components/Camera.jsx

```diff
@@ -72,4 +74,18 @@ class Camera extends Component {
   }
 }
 
+Camera.propTypes = {
+  rotation: PropTypes.string,
```

Comment:
Can this be adjusted so that it's an object containing x, y, and z coords like `position`?